sktime/forecasting/base/adapters/_neuralforecast.py

```diff
@@ -3,13 +3,15 @@
 import abc
 import functools
 import typing
+from inspect import signature
+from logging import warn
```

Comment:
I don't think this is correct. You should throw an actual warning, using `warnings` module or the `sktime` wrapper for the same instead of logging a message at WARNING level.

Use `from sktime.utils.warnings import warn` and change accordingly.